src/Footer/Index.scss

```diff
@@ -0,0 +1,16 @@
+:local {
+  .footer {
+    background: #004c76;
```

Comment:
I recommend setting a foreground color here as well to ensure valid contrast. Did the call outs or UX direction have a value used for most text?

Reply:
This is the same styles (copy & pasted) that were applied to the original terra-ui site footer. This is only displayed on the first Home page. I am open to suggestions on improving this! I might be good to scope these changes in a new issue though?

Reply:
In the interest of getting  a working combined doc site out sooner. I'd like to tackle improving this in another issue.

Reply:
I’m fine with that

Reply:
I’m fine with that

Reply:
I’m fine with that

Reply:
I’m fine with that

Reply:
I’m fine with that